src/Api/Email.php

```diff
@@ -46,4 +48,29 @@ public function send(EmailData $mail): array
 
       return $this->client->httpRequest($path, $body, $headers);
   }
+
+  /**
+   * Send a request to the UniOne API.
+   * @param  array                    $body the request body containing the necessary keys
+   *                                        $body = [
+   *                                        "from_email" => "user@example.com",
+   *                                        "from_name" => "string",
+   *                                        "to_email" => "user@example.com"
+   *                                        ];
+   * @return array                    the response with the status code
+   * @throws GuzzleException
```

Comment:
Why do we have so many exceptions here if request method of Guzzle only has `@throws GuzzleException`?

Reply:
Earlier, the httpRequest method could throw one of them. I deleted the extra ones.